ydb-slo-tests/README.md

```diff
@@ -0,0 +1,205 @@
+# SLO workload
+
+SLO is the type of test where app based on ydb-sdk is tested against falling YDB cluster nodes, tablets, network
+(that is possible situations for distributed DBs with hundreds of nodes)
+
+### Usage:
+
+It has 3 commands:
+
+- `create`  - creates table in database
+- `cleanup` - drops table in database
+- `run`     - runs workload (read and write to table with sets RPS)
+
+[//]: # (### Run examples with all arguments:)
+
+[//]: # ()
+[//]: # (create:)
+
+[//]: # (`python tests/slo/src/ create localhost:2136 /local -t tableName)
```

Comment:
Rewrite the instruction for the rust reality please

Reply:
Python readme.md is outdated.  
See https://github.com/ydb-platform/ydb-python-sdk/blob/main/.github/workflows/slo.yml#L67C1-L68C1

`./tests/slo/src create grpc://localhost:2135 /Root/testdb`